### PR TITLE
[READY] update ebook popup

### LINKED ISSUE #344: Pop-up still too big for some screens
Fix it with a media query based on height

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] hide popup on small screens
- [x] fix scss linter issues (don't use id and color literals in scss)
- [x] ~~close popup on clicking anywhere else but the popup~~ (removed)
- [x] focus first (name) field of popup form on click

Fixes #344

### CODE REVIEW

**Comment by sn3p** [comment]
Would be nice to provide screenshots for visual changes. This way everyone can discuss without having to checkout.

File: source/javascripts/_popup.js [hunk 1]
```diff
@@ -44,32 +45,45 @@ Defacto.popup = {
     }
 
     // Remove popups the user has closed
+    var popupCookie = Defacto.popup.cookies[Defacto.popup.cookiePrefix + this.id];
```
Comment (sn3p) [inline source/javascripts/_popup.js]: `this.id` only works inside the each loop.

**Comment by sn3p** [comment]
Looks good 👍  made some small changes, can you test and lemme know what you think.
Other that that, GTM!

**Comment by RonaldDijkstra** [comment]
No we do not.

File: source/stylesheets/components/_popup.scss [hunk 2]
```diff
@@ -37,6 +40,10 @@ $popup-mobile-mode: max-width 400px;
   width: 100%;
   z-index: 9999999999;
 
+  @include media($popup-hide-mode) {
+    display: none !important;
```
Comment (RonaldDijkstra) [inline source/stylesheets/components/_popup.scss]: Need important to hide the popup, maybe remove it in the future, but not now.

**Comment by RonaldDijkstra** [comment]
Click anywhere for closing was an extra method because on some smaller screens the close button was not visible. With the changes in this PR, the close button is always there or the popup is hidden. 

So.. Ready once again.

**Pull request merged**